lib/web_ui/test/engine/semantics/text_field_test.dart

```diff
@@ -159,7 +159,6 @@ void testMain() {
       expect(domDocument.activeElement, flutterViewEmbedder.glassPaneElement);
       expect(appHostNode.activeElement, strategy.domElement);
       expect(textField.editableElement, strategy.domElement);
-      expect((textField.editableElement as dynamic).value, 'hello');
```

Comment:
This expectation used to verify that the semantics text value gets updated when the framework begins editing a text field. I think to cover that with the new approach we need a new test (or maybe just add it to this test) that sends the text editing state through the text input channel and checks that the editing state is applied to `textField.editableElement`.

Reply:
This is a good point.  I wrote it this way because we're delegating the responsibility of editing state updates to `text_editing.dart`, so it seems like the best place to test framework -> engine editing state updates would be in `text_editing_test.dart` which has existing tests that verify that behavior. 

But is the idea here that we want to explicitly test whether that behavior applies specifically to semantic nodes?

Reply:
I think we want to test the behavior of `text_editing.dart` when using a semantics editing strategy to make sure that the element's value is still being updated correctly.